### DISCUSSION TITLE
underflow encountered in true_divide error

### DISCUSSION THREAD

**Discussion by ankita2020** [bokeh/bokeh#13687] [Internals and design (Q&A)]
I have a dataset with 6439 rows and 28 columns(5 string columns,1 datetime column,3 integers,19 float columns)

These are the installed requirements for `python 3.8.18`
```
bokeh 2.4.3
holoviews 1.14.9
hvplot 0.7.3
panel 0.14.4
```
When trying to create autoviz charts then it is creating only `data quality report`, `displots_nums.html`, and `pair_scatters.html` charts and gives the following error
```
python3.8/site-packages/holoviews/plotting/bokeh/stats.py", line 456, in _kde_data
    ys = (ys/np.nanmax(np.abs(ys)))*(self.violin_width/scale) if len(ys) else []
FloatingPointError: underflow encountered in true_divide
```
when taking only 2000 rows of the same dataset, everything works fine.

**Comment by bryevdv** [comment]
@ankita2020 That error is coming form Holoviews code, not Bokeh. 

> python3.8/site-packages/***holoviews***/plotting/bokeh/stats.py", line 456, in **_kde_data**

Holoviews uses Bokeh internally, but it is a different project, maintained by different people (i.e. *not* by the Bokeh team). I'd suggest you open an issue on one of the Holoviz community resources (discourse, issue tracker, etc).

**Reply by ankita2020** [reply]
@bryevdv Thanks for your response! I've created an issue on the Holoviz community.
